ngraph/core/include/ngraph/util.hpp

```diff
@@ -363,6 +363,15 @@ namespace ngraph
 
         return static_cast<T>(x);
     }
+
+    /// \brief Extracts the tensor data and returns a set of normalized axes created out of it.
+    ///
+    /// \param tensor A pointer to a HostTensor object containing the raw axes data
+    /// \param rank Rank of an operator's input data tensor (used to normalize the axes)
+    /// \param node_description An identifier of the operator's node (used to report errors)
+    AxisSet get_axes_from_tensor(const HostTensorPtr tensor,
```

Comment:
Do we need to have this method in public API? We don't export it and it uses only in one place.

Reply:
I think it will be useful for other ops that can take axes as inputs. For this PR I can move it somewhere else but eventually it will have to be accessible by multiple ops. Any suggestions?

Reply:
I propose, if it is possible, to move this method to header in the source folder. It should avoid additional methods in public API  (moreover we don't export this method).
I understand that the nGraph API still have a lot of internal "public" methods (which are not exported) but I hope we will fix it.